doc/code/qml_pauli.rst

```diff
@@ -74,6 +73,32 @@ array([[ 0. +0.j,  0. +0.j,  0.5+0.j,  0. +0.j],
        [-0.5+0.j,  0. +0.j,  0. +0.j,  0. +0.j],
        [ 0. +0.j,  0.5+0.j,  0. +0.j,  0. +0.j]])
 
+We can intuitively use Pauli arithmetic to construct Hamiltonians consisting of :class:`~pennylane.pauli.PauliWord`
+and :class:`~pennylane.pauli.PauliSentence` objects like the spin-1/2 XXZ model Hamiltonian,
+
+.. math:: H_\text{XXZ} = \sum_j [J^\bot (X_j X_{j+1} + Y_j Y_{j+1}) + J^\text{ZZ} Z_j Z_{j+1} + h Z_j].
+
+Here we look at the simple topology of a one-dimensional chain with periodic boundary conditions (i.e. qubit number :math:`n+1 \equiv 1`).
```

Comment:
Just a preference but since if you print `H` it uses wires [0, 1, 2, 3] instead of [1, 2, 3, 4], I think the boundary condition makes more sense to express as $n \equiv 0$.

Reply:
Sounds good ✅